crates/net/network/src/config.rs

```diff
@@ -307,8 +307,9 @@ impl NetworkConfigBuilder {
         let head = head.unwrap_or(Head {
             hash: chain_spec.genesis_hash(),
             number: 0,
-            total_difficulty: U256::ZERO,
             timestamp: 0,
+            difficulty: chain_spec.genesis.difficulty,
+            total_difficulty: U256::ZERO,
```

Comment:
I think this ultimately won't really matter as we'll override it inside of the session? cc @mattsse 

https://github.com/paradigmxyz/reth/blob/4b0acce9577f6e518d1d7f192536c6ada9a07e95/crates/net/network/src/session/mod.rs#L182-L187

```suggestion
            total_difficulty: chain_spec.genesis.difficulty,
```